### PR TITLE
feat: add Event Types Chips

### LINKED ISSUE #1561: Add Event types to Chips
**Describe the feature you'd like**
To quickly navigate between event types there should be event type chips along with the event date chips

**Would you like to work on the issue?**
- [✓] Yes

### PR DESCRIPTION
**Pull request by aggarwalpulkit596**
Fixes #1561 

Changes: Added event types to result fragment chips

Screenshots for the change:
videotogif_2019 04 10_22 31 04

### CODE REVIEW

**Comment by aggarwalpulkit596** [comment]
@liveHarshit @iamareebjamal please review

**Comment by aggarwalpulkit596** [comment]
@nikit19 review this one also

File: app/src/main/java/org/fossasia/openevent/general/search/SearchResultsFragment.kt [hunk 1]
```diff
@@ -149,19 +133,68 @@ class SearchResultsFragment : Fragment() {
         return rootView
     }
 
+    private fun setChips(date: String = eventDate, type: String = eventType) {
+        if (rootView.chipGroup.childCount>0) {
```
Comment (nikit19) [inline app/src/main/java/org/fossasia/openevent/general/search/SearchResultsFragment.kt]: indentation

File: app/src/main/java/org/fossasia/openevent/general/event/types/EventType.kt [hunk 2]
```diff
@@ -11,7 +11,7 @@ import com.github.jasminb.jsonapi.annotations.Type
 data class EventType(
     @Id(LongIdHandler::class)
     @PrimaryKey
-    val id: Long?,
-    val name: String?,
-    val slug: String?
+    val id: Long,
+    val name: String,
+    val slug: String
```
Comment (nikit19) [inline app/src/main/java/org/fossasia/openevent/general/event/types/EventType.kt]: why? These values will never be null?

Comment (aggarwalpulkit596) [reply app/src/main/java/org/fossasia/openevent/general/event/types/EventType.kt]: will fix them in #1584

Comment (aggarwalpulkit596) [reply app/src/main/java/org/fossasia/openevent/general/event/types/EventType.kt]: yes JSON-API id's can never be null and neither can name or slug

**Comment by aggarwalpulkit596** [comment]
@liveHarshit @iamareebjamal anything left out ?

**Pull request merged**